### PR TITLE
M05diss

### PR DESCRIPTION
**Pull request by wh01s7**
I have made some modifications that have accumulated while using it. Most of them have been discussed.

### CODE REVIEW

File: 2.0/0x100-General/0x104-G4-Business-Logic.md [hunk 1]
```diff
@@ -21,14 +21,15 @@ Category “G4” lists requirements related to the business logic of the smart
...
 | **G4.7** | Verify that the contract uses mechanisms that mitigate transaction-ordering dependence (front-running) attacks (e.g. pre-commit scheme). | 
 | **G4.8** | Verify that the contract does not send funds automatically, but it lets users withdraw funds on their own in the separate transaction instead. |
 | **G4.9** | Verify that the functions are built symmetrically (e.g. in the case of a *deposit*, there is also a *withdraw*). |
 | **G4.10** | Verify that calling a function once with value XY gives a similar result as calling it X times with value Y. |
 | **G4.11** | Verify that the global state is also updated when working on a copy of some data for optimization reasons. |
 | **G4.12** | Verify that when accepting ETH deposits directly and indirectly via WETH, you are validating the case of msg.value > 0 and the amount of WETH transferred in the same function call. |
 | **G4.13** | Verify that there are no off-by one errors, especially with *>=*, *<=* and *length - 1*. |
+| **G4.14** | Verify that string length in ABI decoded data is the same as declared one and there is an upper limit on the length of a string that can be passed to prevent run out of gas. |
```
Comment (damianrusinek) [inline 2.0/0x100-General/0x104-G4-Business-Logic.md:32]: I would add that reference for this check to allow to clarify it.

File: 2.0/0x100-General/0x111-G11-Code-Clarity.md [hunk 2]
```diff
@@ -21,12 +21,13 @@ Category “G11” lists requirements related to the code clarity of the smart c
 | **G11.5** | Verify that the contract uses existing and tested code (e.g. token contract or mechanisms like *ownable*) instead of implementing its own. | 
 | **G11.6** | Verify that the same rules for variable naming are followed throughout all the contracts (e.g. use the same variable name for the same object). | 
 | **G11.7** | Verify that variables with similar names are not used. | 
-| **G11.8** | Verify that all storage variables are initialized. |
```
Comment (damianrusinek) [inline 2.0/0x100-General/0x111-G11-Code-Clarity.md:24]: Are we having a gap in numbers?

File: README.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ Security, Composability, and Transparency are fundamentals of the SCSVS. These v
 * **Integrations** - components with which the project integrates, general recommendations and threats to frequently used smart contracts.
 
 ## How to use SCSVSv2
+![SCSVS_v2_example_checks](https://github.com/wh01s7/SCSVS/assets/39416437/dbe717dd-2679-4d0a-916f-4f2d39509cd2)
```
Comment (damianrusinek) [inline README.md]: Add images to repo and update link.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Smart Contract Security Verification Standard  🚀
-
+![SCSVS_v2_example](https://github.com/wh01s7/SCSVS/assets/39416437/c70fa42f-d013-43a7-8733-33fb79ca3d95)
```
Comment (damianrusinek) [inline README.md]: Add images to repo and update link.

File: 2.0/0x200-Components/0x208-C8-Liquidity-pool.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# C8: Liquidity pool
```
Comment (damianrusinek) [inline 2.0/0x200-Components/0x208-C8-Liquidity-pool.md:1]: Add it on the main page to the list (README.md).

**Review by damianrusinek — CHANGES_REQUESTED** [review]

File: 2.0/0x100-General/0x111-G11-Code-Clarity.md (hunk 2 above)
Comment (wh01s7) [reply 2.0/0x100-General/0x111-G11-Code-Clarity.md:24]: Numbers updated.

File: 2.0/0x100-General/0x104-G4-Business-Logic.md (hunk 1 above)
Comment (wh01s7) [reply 2.0/0x100-General/0x104-G4-Business-Logic.md:32]: Reference added.

File: README.md (hunk 3 above)
Comment (wh01s7) [reply README.md]: PNG added, path changed.

File: README.md (hunk 4 above)
Comment (wh01s7) [reply README.md]: PNG added, path changed.

File: 2.0/0x200-Components/0x208-C8-Liquidity-pool.md (hunk 5 above)
Comment (wh01s7) [reply 2.0/0x200-Components/0x208-C8-Liquidity-pool.md:1]: Added.

**Review by damianrusinek — APPROVED** [review]

**Pull request merged**